state/proxy/logger.go

```diff
@@ -296,6 +298,20 @@ func (s *loggingVmStateDb) GetLogs(hash common.Hash, block uint64, blockHash com
 	return res
 }
 
+// PointCache returns the point cache used in computations.
+func (s *loggingVmStateDb) PointCache() *utils.PointCache {
+	res := s.db.PointCache()
+	s.writeLog("PointCache, %v", res)
+	return res
+}
+
+// Witness retrieves the current state witness.
+func (s *loggingVmStateDb) Witness() *stateless.Witness {
+	res := s.db.Witness()
+	s.writeLog("Witness, %v", res)
```

Comment:
```suggestion
	s.writeLog("Witness, %s", res)
```

`Witness` has [String()](https://github.com/ethereum/go-ethereum/blob/ea3b5095f439d63e35c8c37941836f4815fb380a/core/stateless/witness.go#L122) method